src/base-api-client.ts

```diff
@@ -59,6 +65,47 @@ export abstract class BaseApiClient<TJob, TResult> {
             { 'Content-Type': 'application/json' }, 'json', options);
     }
 
+    /**
+     * Submit audio data for job submission.
+     * @param audioData Audio data to be submitted for job.
+     * @param filename (optional) Name of file associated with audio.
+     * @param options (optional) Options submitted with the job.
+     * @returns Details of submitted job
+     */
+     async _submitJobAudioData(
```

Comment:
```suggestion
     protected async _submitJobAudioData(
```
Same for the other method.

I'm generally a little concerned about these being on the base-api-client because it means the methods are available to the Topic and Sentiment clients as well which allows users to misuse them

Reply:
I can move them to language-id-client

Reply:
are they still access-able if they are marked as protected? we can do `topicClient._submitJobAudioData()`?

Reply:
are they still access-able if they are marked as protected? we can do `topicClient._submitJobAudioData()`?

Reply:
I don't think they would be since only classes that inherit base-api-client could access them. But I moved the methods to language-id-client anyways